enhancements/console/pinned-resources.md

```diff
@@ -0,0 +1,179 @@
+---
+title: pinned-resources
+authors:
+  - "@debsmita1"
+reviewers:
+  - "@jerolimov"
+  - "@JoelSpeed"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2022-11-15
+last-updated: 2022-11-15
+tracking-link:
+  - https://issues.redhat.com/browse/ODC-5012
+status: implementable
+---
+
+# Define pre-pinned resources
+
+## Summary
+
+The Developer perspective on the OpenShift Console provides pinned resources on the navigation which enables the user to go to the resource’s list page with a single click.
+
+Some customers want to provide defaults for those pre-pinned resources for all users.
+
+On the Developer perspective navigation, users can see the default pinned resources if they have not customized these pinned resources. The customers have no way to set these default pinned resources for their users.
+
+This enhancement allows cluster admins to define pre-pinned resources for new users and users who have not customized their navigation items.
+
+![Default pinned resources](default-pinned-resources.png)
+
+## Motivation
+
+Some customers want to provide default pre-pinned resources for all users.
+
+### User Stories
+
+#### Story 1
+
+Customer wants to customize the developer perspective navigation by providing pre-pinned resources to users who do not have their navigation items customized.
+
+As a cluster admin, I want to provide default pre-pinned resources for all users.
```

Comment:
Your user stories are missing the ending.

Normally, user stories are of the form "As a <person>, I want to <do something>, so that <I can achieve some goal>", currently the goal part is missing here, you are telling me what they want to do, but not why, and the why is important to help me understand why you are implementing the feature.